lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -51,16 +51,18 @@ use Net::Async::Blockchain::Client::Websocket;
 use parent qw(Net::Async::Blockchain);
 
 use constant {
-    TRANSFER_SIGNATURE     => '0x' . keccak_256_hex('transfer(address,uint256)'),
-    SYMBOL_SIGNATURE       => '0x' . keccak_256_hex('symbol()'),
-    DECIMALS_SIGNATURE     => '0x' . keccak_256_hex('decimals()'),
     DEFAULT_CURRENCY       => 'ETH',
     DEFAULT_DECIMAL_PLACES => 18,
     UPDATE_ACCOUNTS        => 10,
 };
 
 my %subscription_dictionary = ('transactions' => 'newHeads');
 
+my $filter = Module::PluginFinder->new(
+    search_path => 'Net::Async::Blockchain::Plugins::ETH',
+    filter      => sub { },
```

Comment:
Even I'm not using and the method `modules` does not use this at all, this is required for `Module::PluginFinder` otherwise I will reach this [error message](https://metacpan.org/release/Module-PluginFinder/source/lib/Module/PluginFinder.pm#L145)